NitroxPatcher/Patches/Client/CyclopsMotorMode_SaveEngineStateAndPowerDown_Patch.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Harmony;
+using NitroxModel.Helper;
+using System;
+using System.Reflection;
+
+namespace NitroxPatcher.Patches.Client
+{
+    public class CyclopsMotorMode_SaveEngineStateAndPowerDown_Patch : NitroxPatch
+    {
+        public static readonly Type TARGET_CLASS = typeof(CyclopsMotorMode);
+        public static readonly MethodInfo TARGET_METHOD = TARGET_CLASS.GetMethod("SaveEngineStateAndPowerDown", BindingFlags.Public | BindingFlags.Instance);
+
+        public static bool Prefix(CyclopsMotorMode __instance)
+        {
+            __instance.ReflectionSet("engineOnOldState", __instance.engineOn);
+            return false;
```

Comment:
Why are you unconditionally blocking `ChangeEngineState` here? This function doesn't seem to do anything out of the ordinary, so an explanatory comment would be really nice (add the comment in the code so that future readers don't have to search through the PR's).